gerrymandering/gerrymandering.ipynb

```diff
@@ -192,7 +188,7 @@
     "* Pan/zoom: Select the \"wheel zoom\" tool at the left, and you can zoom in on any region of interest using your scroll wheel.  The shapes should update immediately, while the map tiles will update as soon as they are loaded from the external server, and the racial data will be updated once it has been rendered for the current viewport by datashader.  This behavior is the default for any HoloViews plot using a Bokeh backend.\n",
     "* Tapping: click on any region of the USA and the Congressional district for that region will be highlighted (and the rest dimmed).  This behavior was enabled for the shape outlines by specifying the \"tap\" tool in the options above.\n",
```

Comment:
I'm not sure the tap tool was enabled here. You can either delete this paragraph or add the tap tool.
```suggestion
    "* Tapping: click on any region of the USA and the Congressional district for that region will be highlighted (and the rest dimmed).  This behavior was enabled for the shape outlines by specifying the \"tap\" tool in the options above.\n",
```

Reply:
Please do enable the tap tool to support this behavior.